media-video/haruna/haruna-0.12.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg kde.org ecm
+
+MY_P="${PN}-v${PV}"
+
+DESCRIPTION="Video player built with Qt/QML on top of libmpv"
+HOMEPAGE="https://invent.kde.org/multimedia/haruna"
+SRC_URI="https://invent.kde.org/multimedia/${PN}/-/archive/v${PV}/${MY_P}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+IUSE="youtube"
+
+RDEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtquickcontrols:5
+	dev-qt/qtquickcontrols2:5
+	dev-qt/qtxml:5
+	dev-qt/qtx11extras:5
```

Comment:
keep them sorted please. dev-qt/qtcore can be dropped, it is always provided by ecm.eclass.